### DISCUSSION TITLE
[question] How to set font fallback order of classic UI

### DISCUSSION THREAD

**Discussion by Piroro-hs** [fcitx/fcitx5#630] [Q&A]
I want classic UI to show (at least) Japanese and Emojis. I set up fontconfig to first match `Twemoji Mozilla` and then try `源ノ角ゴシック` (Adobe variant of Noto Sans CJK) on `sans-serif`, classic UI font to`Sans 10`, but it does not show emojis in candidate list.
I tested on QT6 based applications and GTK3 based applications.

Thanks.

**Comment by wengxt** [comment]
Check fontconfig related things, fcitx doesn't do anything special here.

At least I can assure that fcitx5 is capable for displaying such pixmap emoji.
![图片](https://user-images.githubusercontent.com/259684/199131034-469692f1-947a-4fb8-95d0-e6928b3e9992.png)
